src/logic/wallets/getWeb3.ts

```diff
@@ -99,7 +99,6 @@ export const getProviderInfo = async (
   web3Provider: string | Provider,
   providerName = 'Wallet',
 ): Promise<ProviderProps> => {
-  web3 = new Web3(web3Provider)
   const account = await getAccountFrom(web3)
```

Comment:
would it be possible to still pass web3 instead of using a global module variable?

Reply:
Yes, if it's more explicit it's not a problem. Anyway we are always using a getter to get that web3 instance (from the same file)